articles/manual/basics.md

```diff
@@ -1,22 +1,15 @@
 ---
-# Mandatory fields. See more on aka.ms/skyeye/meta.
-title: Quantum development techniques introduction | Microsoft Docs 
-description: Quantum development techniques introduction
-author: QuantumWriter
-ms.author: Christopher.Granade@microsoft.com
-ms.date: 12/11/2017
+title: The basics of Q# | Microsoft Docs 
```

Comment:
It'd be good to check on the preview site, but I don't think we need the `| Microsoft Docs` part now that that's in the global metadata.

Reply:
Done!